### ISSUE TITLE
Kubernetes example

### ISSUE THREAD

**Issue by vikramkhatri** [hashicorp/consul-demo-traffic-splitting#1]
Are you planning to release this example for Kubernetes?

**Comment by nicholasjackson** [comment]
Hi @vikramkhatri I do have a K8s example for this, let me add this to a folder.

**Comment by PaoloAngiolillo** [comment]
@nicholasjackson , hi nicholas, do you still have this k8s example? a colleague and me are trying to work out a demo with consul connect , thank you!

**Comment by nicholasjackson** [comment]
Yes I do @PaoloAngiolillo , let me send you something tomorrow morning.

**Comment by PaoloAngiolillo** [comment]
@nicholasjackson, Thank you!

**Comment by arnab21** [comment]
@nicholasjackson , Hi We are evaluating consul for a cloud platform mesh. Could  you please share/upload the Kubernetes sample
